scripts/contacts-book.js

```diff
@@ -91,7 +90,7 @@ export async function removeContact(account, pubkey) {
 
 /**
  * Render an Account's contact list
- * @param {Account} account
+ * @param {import('.7accounts.js').Account} account
```

Comment:
Oops, quick typo spotted.
```suggestion
 * @param {import('.accounts.js').Account} account
```